### DISCUSSION TITLE
how does the template work with ORCID ids?

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#176] [Q&A] [answered]
If I enter the ORCID id of a user, should manubot return all the journal articles associated with that id? 
I'm trying to avoid adding them one at a time. 
id: ORCID:0000-0003-1483-4236

Thanks, 
Meg

**Comment by vincerubinetti** [comment, marked as answer]
Hi Meg,

First off, while the template _uses_ Manubot, that is only one piece of the "cite process" that the template does. Manubot can only cite one source (paper/book/etc) at a time. For things like ORCID, the template has to first "expand" it into a list of individual sources which then get passed to Manubot.

Here are some documentation pages depending on which version of the template you're using:

Version 1+ docs: https://greene-lab.gitbook.io/lab-website-template-docs/basics/citations (has a more clear diagram of how the cite process works)
Version < 1 docs: https://github.com/greenelab/lab-website-template/wiki/Citations

In both cases, what you want to do is create an `orcid.yaml` file in `/_data` like this:

```yaml
orcid: 0000-0003-1483-4236
```

And if you're using the old version of the template, you'll also have to edit `_config.yaml` as described in the doc link above.

After that, the next time your cite process runs, the template will generate a list of papers for that ORCID and generate full citation details for all of them automatically. You can then use the citation and list components to display the citations wherever and however you want on your site.
